bin/ch-test

```diff
@@ -786,8 +786,7 @@ printf 'contributor:  %s\n\n' "$ch_contributor_note"
 
 printf "%-21s %s" 'phase:' "$phase"
 if [[ -z $phase ]]; then
-    phase=all
-    printf 'build run examples'
+    fatal 'phase: no phase specified'
```

Comment:
Should this still print `build run examples` if you give `all`?

Reply:
I think so. I feel `build run examples` is more explicit than `all`, however, I am wiling to change it if you prefer.

Reply:
I agree; I think `build run examples` is clearer. But in that case, looks like that code needs to be added.

Reply:
We added `all` to the above case statement that assigns `phase=all`. Down further our primary switch has:

```
    all)
        phase=build
        dir_tar_mk
        builder_check
        test_make
        test_build

        phase=run
        dir_img_mk
        dir_tar_check
        test_make
        test_run

        phase=examples
        dir_tar_check
        test_make
        test_examples
        win
```

Am I missing something?

Reply:
Looks like I need to update the docs for starters (oops).

Reply:
I think I made a mistake. The current behavior in this PR with phase `all` is that it rolls through `build`, `run`, `examples`, announcing each in turn? Right? If so, then my objection is erroneous because it *does* announce each of the phases, just not where I was looking.